apps/src/templates/progress/ProgressDetailToggle.story.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ProgressDetailToggle from './ProgressDetailToggle';
+import { Provider } from 'react-redux';
+import {combineReducers, createStore} from 'redux';
+import progress, { setIsSummaryView } from '@cdo/apps/code-studio/progressRedux';
+
+export default storybook => {
+  const storyTable = [];
+
+  // Stick these in anonymous functions so that each instance gets its own
+  // store
+  (function () {
+    const store = createStore(combineReducers({progress}));
+    storyTable.push({
+      name:'isSummary is true',
+      story: () => (
+        <Provider store={store}>
+          <ProgressDetailToggle/>
+        </Provider>
```

Comment:
Instead of wrapping these components in a `Provider`, you can also just pass a `store` prop directly to them:

```jsx
<ProgressDetailToggle store={store}/>
```